cmd/manager/main.go

```diff
@@ -117,6 +123,14 @@ func main() {
 
 	setupLog.Info("starting up the controller", "version info", version.String())
 
+	var globalPullSecretKey *k8stypes.NamespacedName
+	if globalPullSecret != "" {
+		secretParts := strings.Split(globalPullSecret, "/")
+		globalPullSecretKey = &k8stypes.NamespacedName{Name: secretParts[1], Namespace: secretParts[0]}
+	} else {
+		globalPullSecretKey = nil
+	}
```

Comment:
This is unnecessary and can be deleted. `nil` is the default value of a pointer that gets assigned in line 126 when you declare the variable.

Reply:
right, I was using `var globalPullSecretKey string` before I decided to switch to a pointer, so remnant of that. Removing it.